perun-core/src/main/java/cz/metacentrum/perun/core/provisioning/HierarchicalHashedDataGenerator.java

```diff
@@ -54,6 +56,7 @@
 	private final Service service;
 	private final Facility facility;
 	private final GenDataProvider dataProvider;
+	private final Set<Member> allMembers = new HashSet<>();
```

Comment:
Shouldn't ``hashes: [...hashes...]`` be removed also from this class javadoc? It was removed from services manager methods API docs.

Reply:
Yes, I have removed it.